src/app/public/modules/flyout/flyout-instance.ts

```diff
@@ -15,19 +15,53 @@ import {
   SkyFlyoutMessageType
 } from './types/flyout-message-type';
 
+/**
+ * The `SkyFlyoutService` launches flyouts and returns a `SkyFlyoutInstance` object to represent
+ * a single displayed flyout.
+ */
 export class SkyFlyoutInstance<T> implements OnDestroy {
+
+  /**
+   * An event that the flyout instance emits when it closes.
+   */
   public closed = new EventEmitter<void>();
+
+  /**
+   * Specifies the instance of the component to display in the flyout.
+   */
   public componentInstance: T;
+
+  /**
+   * Used to communicate with the host component.
+   * @internal
+   */
+  public get hostController(): Subject<SkyFlyoutMessage> {
+    return this._hostController;
+  }
+
+  /**
+   * Indicates if the flyout is open.
```

Comment:
Is this reworded description accurate? I thought that we had it phrased the way we do because the property doesn't actually control whether the flyout opens but rather reports on the flyout's state. I may be misremembering that, but just checking.

Also, in the current wording, our standard wording is `Indicates whether...` instead of `Indicates if...` Superminor, but I'd change it for consistency if we stick with the changed version.